ws/portforward/model.go

```diff
@@ -36,6 +38,9 @@ const (
 	PortForwardProtocolUDP = "udp"
 )
 
+// PropertyConnectionID is the name of the property holding the connection ID
+const PropertyConnectionID = "connection_id"
+
 // PortForwardNew represents a new port forwarding request
 type PortForwardNew struct {
 	RemoteHost *string              `msgpack:"remote_host" json:"remote_host"`
```

Comment:
While you're at it, can you make RemotePort `uint16`?

Reply:
done